src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="backdownField">
+        <legend>Personal information</legend>
+        <label for="sname">Surname:</label>
+        <input
+          class="backdownInput"
+          type="text"
+          name="sname"
+          autocomplete="off"
+        ><br>
+
+        <label for="fname">Name:</label>
+        <input
+          class="backdownInput"
+          type="text"
+          name="fname"
+          autocomplete="off"
+        ><br>
+
+        <label for="age">How old are You?</label>
+        <input
+          class="backdownInput"
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+        ><br>
+
+        <label for="bdate">Full date of birth:</label>
+        <input
+          class="backdownInput"
+          type="date"
+          name="bdate"
+        ><br>
+
+        <label for="agreem">I accept the term of the agreement</label>
+        <input
+          type="checkbox"
```

Comment:
Your label doesn't work without id for input. Fix it everywhere
```suggestion
          type="checkbox"
          id="agreem"
```